presets/4.3/vtx/diatone_mamba_ultra_1000.txt

```diff
@@ -0,0 +1,52 @@
+#$ TITLE: Diatone Mamba Ultra 1000 VTX Tables
+#$ FIRMWARE_VERSION: 4.2
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: VTX
+#$ STATUS: COMMUNITY
+#$ KEYWORDS:  vtx, vtx table, diatone, mamaba, ultra 1000
+#$ AUTHOR: NemesisXB
+#$ DESCRIPTION: VTX tables for the Diatone Mamba Ultra 1000.
+#$ DESCRIPTION: Select the the HAM option if you have a HAM license or the NO HAM if you do not.
+#$ DESCRIPTION: EU users select the NO HAM option.
+#$ DISCLAIMER:  Only select one option. All previous VTX Table settings will be reset.
+#$ INCLUDE_DISCLAIMER: misc/disclaimer/en/vtxtable.txt
+#$ DISCUSSION:  https://github.com/betaflight/firmware-presets/pull/107
+
+# reset basic Vtx settings
```

Comment:
same as other PR please use the default file not manually reseting 

`#$ INCLUDE: presets/4.3/vtx/defaults.txt`